sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Release History
+
+## 1.0.0-preview.4
+
+### Breaking changes
+
+- Made `Keys` and `Labels` in `SettingSelector` read-only.
+- Removed `SetConfigurationSetting` overload that took `MatchCondition` as a parameter.
+- Changed `SettingFields.Locked` to `SettingFields.ReadOnly`.
+- Renamed `AzureClientBuilderExtensions` to `ConfigurationClientBuilderExtensions` in `Microsoft.Extensions.Azure` namespace.
+- Made `ConfigurationClientOptions.Version` property internal.
+- Changed client method names `Add`, `Get`, `Set`, `Delete`, and `GetSettings`, as well as their async versions to `AddConfigurationSetting`, `GetConfigurationSetting`, `SetConfigurationSetting`, `DeleteConfigurationSetting`, and `GetConfigurationSettings` respectively.
+- Changed `ConfigurationSetting.ReadOnly` to `ConfigurationSetting.IsReadOnly`.
+
+### Major changes
```

Comment:
We have also changed `Equals` and `GetHashCode` implementation of several classes.